data/nabadev.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Nabajyoti Bharali",
+  "type": "personal",
+  "bio": "Self-taught web developer",
+  "avatar": "https://github.com/nabadev.png/",
```

Comment:
You can remove this line, it is not required anymore :+1: 
```Suggestion
```

Reply:
Thanks for the information :)